addons/point_of_sale/tools/posbox/configuration/wireless_ap.sh

```diff
@@ -1,82 +0,0 @@
-#!/usr/bin/env bash
-
-function get_conf () {
-    KEY="${1}"
-    CONF_FILE="${2}"
-    # These commands seek for keys on the configuration file,
-    # for each key found, trim spaces and store the corresponding value
-
-    touch "$CONF_FILE"  # create the file if it does not exist
-    awk -v key="$KEY" -F= '$1 ~ "^" key "[[:space:]]*" {gsub(/^[[:space:]]+|[[:space:]]+$/, "", $2); print $2}' "$CONF_FILE"
-}
-
-FORCE_HOST_AP="${1}"
-CONF_FILE="/home/pi/odoo.conf"
-COUNTER=0
-ESSID=$(get_conf "wifi_ssid" "${CONF_FILE}")
-PASSWORD=$(get_conf "wifi_password" "${CONF_FILE}")
-
-# we need to wait to receive an ip address from the dhcp before enable the access point.
-# only if no configuration file for the wifi networks is recorded
-if [ "${ESSID}" ] && [ "${PASSWORD}" ] && [ -z "${FORCE_HOST_AP}" ] ; then
-	while [ "$(hostname -I)" = '' ] && [ "$COUNTER" -le 10 ]; do sleep 2;((COUNTER++)); done
-fi
```

Comment:
I think we need to replicate this functionality (wait 20 seconds for an IP address on boot, then start the hotspot if not). We know from #187862 that without this it can cause the hotspot to start when it shouldn't, stopping the pairing code from appearing.

Reply:
ah yes right, I'll add it